tests/artifacts/configuration.ini

```diff
@@ -1,5 +1,5 @@
 [GENERAL]
-dataset_path = datasets/mnist
+dataset_path = tests/artifacts/runner_scheduler
```

Comment:
Well you're right, this is only going to be used by the runner/scheduler, and I should put this in the corresponding folder.

But I wanted to point out that there shouldn't be a global `configuration.ini`, since pretty much all tests should use their own, or none at all.